packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -252,22 +285,92 @@ class _AddNewProductPageState extends State<AddNewProductPage>
   Widget _buildCard(
     final List<Widget> children,
   ) =>
-      SmoothCard(
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: children,
+      SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(16.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: children,
+          ),
         ),
       );
 
   Attribute? _getAttribute(final String tag) =>
       upToDateProduct.getAttributes(<String>[tag])[tag];
 
+  Widget _backButton() {
+    return Container(
+      margin: const EdgeInsetsDirectional.fromSTEB(10.0, 10.0, 10.0, 0.0),
+      width: 20.0,
+      height: 20.0,
+      child: IconButton(
+        onPressed: () => _onWillPop().then((bool leaveThePage) =>
+            leaveThePage ? Navigator.of(context).pop() : null),
+        alignment: Alignment.center,
+        padding: EdgeInsets.zero,
+        icon: const Icon(Icons.arrow_back),
+      ),
+    );
+  }
+
+  Widget _getButtons() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: <Widget>[
+        ElevatedButton(
+          style: ElevatedButton.styleFrom(
+            minimumSize: Size(MediaQuery.of(context).size.width * 0.35, 40.0),
+            backgroundColor: Theme.of(context).colorScheme.secondary,
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(20.0),
+            ),
+          ),
+          onPressed: () {
+            _onWillPop().then((bool leaveThePage) =>
+                leaveThePage ? Navigator.of(context).pop() : null);
+          },
+          child: Text(AppLocalizations.of(context).cancel,
+              style: const TextStyle(
+                  color: Colors.black,
+                  fontSize: 20.0,
+                  fontWeight: FontWeight.bold)),
+        ),
+        const SizedBox(width: 10.0),
+        ElevatedButton(
+          style: ElevatedButton.styleFrom(
+            minimumSize: Size(MediaQuery.of(context).size.width * 0.35, 40.0),
+            backgroundColor: const Color(0xFF341100),
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(20.0),
+            ),
+          ),
+          onPressed: () {
+            _pageController.nextPage(
+                duration: const Duration(milliseconds: 200),
+                curve: Curves.easeOut);
+          },
+          child: Text(
+            _isLastPage
+                ? AppLocalizations.of(context).finish
+                : AppLocalizations.of(context).next_label,
+            style: const TextStyle(
+                color: Colors.white,
+                fontSize: 20.0,
+                fontWeight: FontWeight.bold),
```

Comment:
Trailing comma.
For the white color, is-it OK in both light and dark modes?

Reply:
Do you want a Trailing comma after `fontWeight: FontWeight.bold` this line?

Yes that is okay in both mode 
Light Mode 
![image](https://github.com/openfoodfacts/smooth-app/assets/86102418/b503ed0e-a1a8-488d-8f9c-e65f85616694)
Dark mode 
![image](https://github.com/openfoodfacts/smooth-app/assets/86102418/e08428ea-8d06-4c01-84d6-9d632c3f5316)